content/tutorials/reactor-event-ifttt.textile

```diff
@@ -0,0 +1,196 @@
+# Add the name for the Webhook event that you created earlier (eg channel_message_published)
+
+Next we will need to choose a service that we want to hook our published message up to. In this case we're going to choose Slack, but you could use any number or services that IFTTT provides.
+
+# Click the "*&plus; That*" which should now be presented to you
+# Search for Slack
+# Click "*Slack*"
+# Click "*Connect*"
+# Log in to your slack workplace in the resulting popup
+# Allow the requested permissions
+
+The final stage of our setup will be choosing an action to trigger in Slack. In this case we want to post to a channel. The IFTTT Slack service allows you to post to channels, to private groups or to send direct messages. For the purpose of this tutorial we'll be sending ourselves private messages (for the sake of not filling your slack channels with messages!)
+
+We'll be sending the body of the message that was published to your Ably channel to slack, so for this tutorial we're only going to send one value as the message body, but you could customise this as much as you like.
+
+# Click "*Post to channel*"
+# In the resulting form, under "Which channnel":
+## select Direct Messages 
+## select your own Slack handle
+# Under "Message":
+## delete the content in the textbox 
+## Click "*Add ingredient*"
+## Add "Value1"
+# Under "Title URL" add "Captain Picard" (or the captain name of your choice!)
+# You can even add a thumbnail avatar of your captain if you'd like by pasting a url to your chosen image
+# Click "*Create Action*"
+# Click "*Finish*"
+
+You have now set up your Ably channel to post to Slack using IFTTT! 
+
+In the next section we'll create a game which will publish messages to our Ably channel, thereby triggering a Slack message to be sent.
+
+h2. Building the Game
+
+To get you up and running quickly, we've made the code for the game available via "Glitch":https://glitch.com/~ably-space-game. You can remix or clone this repository. Press the *Remix to Edit* button in the top right hand corner of the page. We'll need to edit the code a little to add in your Ably API key.
+
+h2. How the game works
+
+The game is a web app, created with "Glitch":https://glitch.com/. It uses "Node":https://nodejs.org/en/ and "Express":https://expressjs.com/. Instructions on how the game works can be read at in the "readme.md":https://glitch.com/edit/#!/ably-space-game?path=README.md:1:0. We're using the game to generate data to send to our Ably channel, if you already have data going to an Ably channel then you will not need the game at all! The game generates 10 'moves' for the player to take and publishes them, one by one, to an Ably channel as the player completes previous moves.
+
+h2. How does the game publish messages?
+
+The game publishes message "commands" to an Ably channel. Take a look at the *ablyConnector.js* file in the root, this uses the Ably REST API to publish to an Ably channel. You'll see some code like this
```

Comment:
You refer to Ably Channel before, and Ably channel here. Please aim to be consistent.

Reply:
> Take a look at the *ablyConnector.js* file in the root

Can we not link to that file from here?